### PR TITLE
seo: Create canonical link in headers

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
This removes the duplicate pages from search engines and helps getting more people to find FlowFuse Dashboard.

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: docs/.vitepress/config.js [hunk 1]
```diff
@@ -37,6 +37,17 @@ export default ({ mode }) => {
         "window.dataLayer = window.dataLayer || [];\nfunction gtag(){dataLayer.push(arguments);}\ngtag('js', new Date());\ngtag('config', 'G-MNGVF5NCF7');",
       ],
     ],
+    transformPageData(pageData) {
```
Comment (ZJvandeWeg) [inline docs/.vitepress/config.js:40]: The kicker of my issues: You can have a hash where a function is a key and value at the same time 🤯

Comment (joepavitt) [reply docs/.vitepress/config.js:40]: Yep, this is shorthand for `transformPageData: transformPageData(pageData) {`. It can be done with objects too.

**Comment by joepavitt** [comment]
Out of interest - which pages were flagging as duplicates?

**Comment by ZJvandeWeg** [comment]
@joepavitt 
Screenshot 2024-02-06 at 10 47 47

**Review by joepavitt — APPROVED** [review]

**Pull request merged**